src/commands.rs

```diff
@@ -17,3 +22,28 @@ pub(crate) mod upgrade;
 
 #[cfg(feature = "pczt-qr")]
 pub(crate) mod keystone;
+
+pub(crate) fn select_account<DbT: WalletRead<AccountId = AccountUuid>>(
+    db_data: &DbT,
+    account_uuid: Option<Uuid>,
+) -> Result<DbT::Account, anyhow::Error>
+where
+    DbT::Error: std::error::Error + Sync + Send + 'static,
+{
+    let account_id = match account_uuid {
+        Some(uuid) => Ok(AccountUuid::from_uuid(uuid)),
+        None => {
+            let account_ids = db_data.get_account_ids()?;
+            account_ids.first().map_or(
+                Err(anyhow!(
+                    "More than one account is available; please specify the account UUID."
```

Comment:
This seems like it's doing the opposite: if `.first()` returns `None` then there are _no_ accounts, not _several_.

Reply:
:facepalm: I had written this to check `len` and call `first`, and then I was like "oh but I need to error anyway" and forgot to reintroduce the length check.